### PR TITLE
More robustification for mixture model tests

### PR DESCRIPTION
**Pull request by lindsayad**
Follow-on to https://github.com/idaholab/moose/pull/26658

### CODE REVIEW

File: modules/navier_stokes/test/tests/finite_volume/two_phase/mixture_model/channel-advection-slip.i [hunk 1]
```diff
@@ -226,10 +237,12 @@ velocity_interp_method = 'rc'
     rho_d = ${rho_d}
     particle_diameter = ${dp}
     linear_coef_name = 'Darcy_coefficient'
+    outputs = 'out'
+    output_properties = 'vel_slip_x'
```
Comment (GiudGiud) [inline modules/navier_stokes/test/tests/finite_volume/two_phase/mixture_model/channel-advection-slip.i:241]: noice

**Review by GiudGiud — APPROVED** [review]
looks good.

File: modules/navier_stokes/test/tests/finite_volume/two_phase/mixture_model/channel-advection-slip.i (hunk 1 above)
Comment (lindsayad) [reply modules/navier_stokes/test/tests/finite_volume/two_phase/mixture_model/channel-advection-slip.i:241]: haha I was excited to use your new capability!

**Review by tanoret — APPROVED** [review]
All good. Nice changes!

We will need the AuxKernels at some point for the linear solvers, but we can add them back later.

**Comment by lindsayad** [comment]
failures are unrelated

**Pull request merged**